Sources/AppStoreConnectCLI/Services/Operations/ModifyBetaGroupOperation.swift

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ModifyBetaGroupOperation: APIOperation {
+    struct Options {
+        let appId: String
+        let currentGroupName: String
+        let newGroupName: String?
+        let publicLinkEnabled: Bool?
+        let publicLinkLimit: Int?
+        let publicLinkLimitEnabled: Bool?
+    }
+
+    enum Error: LocalizedError {
+        case betaGroupNotFound(groupName: String, appId: String)
+        case betaGroupNotUniqueToApp(groupName: String, appId: String)
+
+        var errorDescription: String? {
+            switch self {
+            case .betaGroupNotFound(let groupName, let appId):
+                return "No beta group found with name: \(groupName) and app id: \(appId)"
```

Comment:
Showing appId in the error message is alright. But actually what the user input is the `bundleId`, so it might be a little inconsistent here.

Reply:
I think you're right, maybe I'll pass in the whole app